e2e/ethdevnet.go

```diff
@@ -0,0 +1,40 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/fakebeacon"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/geth"
+	"github.com/ethereum-optimism/optimism/op-service/clock"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func ethdevnet(_ context.Context, blockTime uint64, genesis *core.Genesis) (*rpc.Client, string, error) {
+	now := time.Now().Unix()
+	blobsDirectory := filepath.Join("artifacts", "blobs")
+
+	beacon := fakebeacon.NewBeacon(nil, blobsDirectory, uint64(now), blockTime)
```

Comment:
For my own understanding - could `genesis.Timestamp` be used instead of `time.Now().Unix()` for the beacon block genesis time?

Reply:
I checked. It can, and probably should, and now does. 5876226